### PR TITLE
Pull requests api cleanup

### PR DESCRIPTION
**Pull request by bandjeo**

### CODE REVIEW

File: oss-cloud-backend/gitHubApiService.js [hunk 1]
```diff
@@ -48,35 +45,35 @@ module.exports.getParentRepos = async (repos) => {
...
+searchUserPullRequests = async(username, repo) => {
   console.log("searching pull requests: ", username, repo.name)
-  var pulls = []
-  let pullNum = 0;
+  var pullRequests = []
+  let pullRequestNum = 0;
   do {
     await octokit.search.issuesAndPullRequests({
       q: "repo:" + repo.owner.login + "/" + repo.name + "+author:" + username + "+is:pr",
       per_page: 100
```
Comment (mjovanovic) [inline oss-cloud-backend/gitHubApiService.js]: `page` parameter is missing here (https://octokit.github.io/rest.js/#octokit-routes-search-issues-and-pull-requests)

File: oss-cloud-backend/handler.js [hunk 2]
```diff
@@ -22,22 +24,49 @@ module.exports.hello = async event => {
...
-    })
+module.exports.getPullRequests = async (event, context, callback) => {
+  let response;
+  // check if request is valid
+  try {
+    var body = JSON.parse(event.body);
+    body.username;
+  } catch {
+    response = {
+      statusCode: 500,
```
Comment (mjovanovic) [inline oss-cloud-backend/handler.js]: If the JSON format is bad that's a client side error so the status code should be 4XX

File: oss-cloud-backend/gitHubApiService.js [hunk 3]
```diff
@@ -30,7 +27,7 @@ module.exports.getForkedRepos = async(username) => {
 
 // makes a call to github api for each repo in array, returning a more detailed representation of the repo
 module.exports.getRepoDetails = async (repos) => {
```
Comment (banecogic) [inline oss-cloud-backend/gitHubApiService.js]: There is no need for `return await Promise...`.
`await` keyword can be omitted in return statement of `async` function. [Link to StackOverflow](https://stackoverflow.com/questions/38708550/difference-between-return-await-promise-and-return-promise)

File: oss-cloud-backend/handler.js (hunk 2 above)
Comment (bandjeo) [reply oss-cloud-backend/handler.js]: Changed response status to 400.

File: oss-cloud-backend/gitHubApiService.js (hunk 1 above)
Comment (bandjeo) [reply oss-cloud-backend/gitHubApiService.js]: Added page parameter.

**Pull request merged**